spec/movie_reviews_spec.rb

```diff
@@ -0,0 +1,121 @@
+require 'spec_helper'
+
+describe MoviesNYT do
+  context  'Movie Reviews Critics picks' do 
+    before(:all) do 
+      @review_picks = MoviesNYT.new.movie_reviews_nyt
+      @review_picks.reviews('picks')
+    end 
+     
+    it 'Should be an array' do 
```

Comment:
These test descriptions are really hard to follow, you need to specify the things you expect to be something. For example '   it 'should return the thing as an data_type'